Bottom: 93a8442b5d1434b3f2afc21e71210a42ff69d406
Top:    b7f7c90d6559f232e9859c1fe10a1f4d424ea3bd
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2008-10-02 16:38:19 +0100

Refresh of allow-polygons-with-clearlinef

---

diff --git a/src/change.c b/src/change.c
index 92ba535..e0ef99b 100644
--- a/src/change.c
+++ b/src/change.c
@@ -1749,7 +1749,6 @@ ChangeSelectedJoin (int types)
   change = SelectedOperation (&ChangeJoinFunctions, False, types);
   if (change)
     {
-      printf ("Hello\n");
       Draw ();
       IncrementUndoSerialNumber ();
     }
